### PR TITLE
Add johndao to the list of contributors

### PR DESCRIPTION
**Pull request by johndao1005**

### CODE REVIEW

File: Data/member.json [hunk 1]
```diff
@@ -413,6 +413,13 @@
             "description": "Bangladeshi | Studying Computer Science and Engineering at BRAC University, Dhaka | Webmaster, BRACU IEEE SB | Competitive Programmer |",
             "githuburl": "https://github.com/ZuhairHossain/",
             "linkedinurl": "https://www.linkedin.com/in/syed-zuhair-hossain/"
+        },{
```
Comment (NS007-dev) [inline Data/member.json]: Please can you move the brace after the comma to the next line @johndao1005 ? Thank you.

**Review by NS007-dev — CHANGES_REQUESTED** [review]

File: Data/member.json (hunk 1 above)
Comment (johndao1005) [reply Data/member.json]: opps my bad. i will get it fix

**Review by NS007-dev — APPROVED** [review]
Thank you for fixing it! LGTM @johndao1005

**Review by MrSlimCoder — APPROVED** [review]
LGTM

**Comment by MrSlimCoder** [comment]
@all-contributors add @johndao1005 for content.

**Pull request merged**